contracts/consensus-gateway/ConsensusCogateway.sol

```diff
@@ -85,4 +94,65 @@ contract ConsensusCogateway is MasterCopyNonUpgradable, MessageBus, ConsensusGat
             _maxStorageRootItems
         );
     }
+
+    /**
+     * @notice It allows to withdraw utMOST tokens. Withdrawer needs to approve
+     *         ConsensusCoGateway contract for amount to be withdrawn.
+     *
+     * @dev Function requires :
+     *          - Amount must not be 0.
+     *          - Beneficiary must not be 0.
+     *          - Amount must be greater than gas price and gas limit.
```

Comment:
Please update the statement. It is incorrect.

Reply:
Done